test/azuremetadata/generated/Lro/README.md

```diff
@@ -1,80 +1,80 @@
-- node.js version 6.x.x or higher
-- browser javascript
-
-## How to Install
-
-- nodejs
-```
-npm install
-```
-- browser
-```html
-<script type="text/javascript" src="https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/lib/services//autoRestLongRunningOperationTestServiceBundle.js"></script>
-```
-
-## How to use
-
-### nodejs - Authentication, client creation and put200Succeeded lROs as an example written in TypeScript.
-
-```ts
-import * as msRest from "ms-rest-js";
-import * as msRestAzure from "ms-rest-azure-js";
-import * as msRestNodeAuth from "ms-rest-nodeauth";
-import { AutoRestLongRunningOperationTestService, AutoRestLongRunningOperationTestServiceModels, AutoRestLongRunningOperationTestServiceMappers } from "";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new AutoRestLongRunningOperationTestService(creds, subscriptionId);
-  const product: AutoRestLongRunningOperationTestServiceModels.Product = {
-    tags: { "key1": "testtags" },
-    location: "westus",
-    provisioningState: "testprovisioningState"
-  };
-  client.lROs.put200Succeeded(product).then((result) => {
-    console.log("The result is:");
-    console.log(result);
-  });
-}).catch((err) => {
-  console.error(err);
-});
-```
-
-### browser - Authentication, client creation and put200Succeeded lROs as an example written in javascript.
-
-- index.html
-```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <title> sample</title>
-    <script type="text/javascript" src="https://raw.githubusercontent.com/Azure/ms-rest-js/master/msRestBundle.js"></script>
-    <script type="text/javascript" src="https://raw.githubusercontent.com/Azure/ms-rest-js/master/msRestAzureBundle.js"></script>
-    <script type="text/javascript" src="https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/lib/services//autoRestLongRunningOperationTestServiceBundle.js"></script>
-    <script type="text/javascript">
-      const subscriptionId = "<Subscription_Id>";
-      const token = "<access_token>";
-      const creds = new msRest.TokenCredentials(token);
-      const client = new AutoRestLongRunningOperationTestService(creds, undefined, subscriptionId);
-      const product = {
-        tags: { "key1": "testtags" },
-        location: "westus",
-        provisioningState: "testprovisioningState"
-      };
-      client.lROs.put200Succeeded(product).then((result) => {
-        console.log("The result is:");
-        console.log(result);
-      }).catch((err) => {
-        console.log('An error ocurred:');
-        console.error(err);
-      });
-    </script>
-  </head>
-  <body>
-  </body>
-</html>
-```
-
-# Related projects
- - [Microsoft Azure SDK for Javascript](https://github.com/Azure/azure-sdk-for-js)
+# Microsoft Azure SDK for isomorphic javascript - AutoRestLongRunningOperationTestService

```

Comment:
Wish all this line endings junk wasn't in here. But I don't believe meaningful changes happened.